crates/gpui/src/platform/windows/platform.rs

```diff
@@ -300,7 +303,32 @@ impl Platform for WindowsPlatform {
 
     // todo!("windows")
     fn prompt_for_paths(&self, options: PathPromptOptions) -> Receiver<Option<Vec<PathBuf>>> {
-        unimplemented!()
+        let (done_tx, done_rx) = channel();
+        self.foreground_executor()
+            .spawn(async move {
+                let done_tx = Cell::new(Some(done_tx));
+                let mut filename: [u8; 1024] = [0; 1024];
+                let mut open_filename = OPENFILENAMEA  {
+                    lStructSize: std::mem::size_of::<OPENFILENAMEA>() as _,
+                    lpstrFile: windows::core::PSTR::from_raw(filename.as_mut_ptr()),
+                    nMaxFile: 1024,
+                    Flags: OFN_PATHMUSTEXIST | OFN_FILEMUSTEXIST,
+                    ..unsafe { std::mem::zeroed() }
+                };
+                let ok = unsafe { GetOpenFileNameA(&mut open_filename) };
+                let result: Option<Vec<PathBuf>> = if ok.as_bool() {
+                    let filename = unsafe { open_filename.lpstrFile.to_string() }.unwrap();
```

Comment:
This conversion is incorrect. PathBufs can use platform representations of files IIRC, and don't need a round trip through UTF-8

Reply:
Thanks for the feedback. Apart from that problem there are other limitations in this approach. Since there is another a PR already #8919 to solve the same problem I am going to close this PR.